src/Pages/SignIn/SignIn.js

```diff
@@ -54,21 +54,25 @@ const SignIn = ({ loginStatus }) => {
   };
 
   const handleLogin = () => {
-    axios({
-      method: "POST",
-      url: `${API}hrdsuser/login/`,
-      data: {
-        email: inputValue.email,
-        password: inputValue.password,
-      },
-    })
-      .then((res) => {
-        sessionStorage.setItem("access_token", res.data.data.token);
-        setLogin(true);
-      })
-      .then(() => {
-        history.push("/");
-      });
+    const { email, password } = inputValue;
+    !email || !password
+      ? alert("아이디와 비밀번호를 입력해주세요.")
+      : axios({
+          method: "POST",
+          url: `${API}hrdsuser/login/`,
+          data: {
+            email: inputValue.email,
+            password: inputValue.password,
+          },
```

Comment:
```suggestion
          data: {
            email,
            password,
          },
```
위에서 destructuring 했으니 이렇게도 되지 않을까요?

Reply:
여기를 못 봤네요! 감사합니다